gunicorn/util.py

```diff
@@ -357,13 +356,12 @@ def import_app(module):
 
     mod = sys.modules[module]
 
-    is_debug = logging.root.level == logging.DEBUG
     try:
         app = eval(obj, vars(mod))
     except NameError:
-        if is_debug:
-            traceback.print_exception(*sys.exc_info())
-        raise AppImportError("Failed to find application object %r in %r" % (obj, module))
+        raise AppImportError("Failed to find application object '{0}' in '{1}':\n{2}".format(
```

Comment:
You can use `{0!r}` instead of `'{0}'`. But I agree with @tilgovi's comment at https://github.com/benoitc/gunicorn/pull/1821#issuecomment-402309585 that we don't need to change the exception message.